comms-commands/orb.yml

```diff
@@ -58,7 +58,7 @@ jobs:
             - terraform_apply:
                 path: << parameters.path >>
                 run_shipit: false
-                backend_config_file: ./environments/<< parameters.environment >>/<< parameters.environment >>.backend
+                backend_config_file: ../environments/<< parameters.environment >>/<< parameters.environment >>.backend
                 var_file: ./environments/<< parameters.environment >>/<< parameters.environment >>.tfvars
```

Comment:
I think these need to be updated to the new paths as well

Reply:
Oh I think I messed up my rebase :(
Will fix